src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -42,7 +42,7 @@ public function write($messages, $newline = false, $options = 0);
     /**
      * Writes a message to the output and adds a newline at the end.
      *
-     * @param string|iterable $messages The message as an iterable of lines of a single string
+     * @param string|iterable $messages The message as an iterable of lines or a single string
```

Comment:
s/lines/string/ also?

Reply:
In this case the strings are output as lines (printLn). But I don't mind to change it as well if you want.

Reply:
that'd be consistent IMHO

Reply:
I agree with this suggestion too.

Reply:
Re-reading, I'd prefer to keep "lines" above, don't you think?

Reply:
I think I prefer talking about php types only, but no strong opinion really.